### PR TITLE
Update SwanLake Alpha4 release notes for debugger

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
$subject.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-alpha4-release-note.md [hunk 1]
```diff
@@ -105,6 +105,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
 
 #### New Features
 
+- Added debugging support for package dependency sources. With this feature, now you can step-in and debug the sources
+  related to the following dependency package types, that are used within your ballerina program.
+  - language library sources
+  - standard library sources
+  - imported modules from the central
```
Comment (MaryamZi) [inline release-notes/swan-lake-alpha4-release-note.md:112]: Shall we make this a heading instead?

```suggestion
##### Debugging Support for Dependencies

With this feature you can now step-in and debug the sources of the following dependencies used within your Ballerina program.
  - language library
  - standard library
  - modules imported from Ballerina Central
```

Comment (NipunaRanasinghe) [reply release-notes/swan-lake-alpha4-release-note.md:112]: @MaryamZi +1.. shall we also make `language library` -> `language libraries` and `standard library` -> `standard libraries` in your suggestion?

File: release-notes/swan-lake-alpha4-release-note.md [hunk 2]
```diff
@@ -105,6 +105,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
 
 #### New Features
 
+- Added debugging support for package dependency sources. With this feature, now you can step-in and debug the sources
```
Comment (praneesha) [inline release-notes/swan-lake-alpha4-release-note.md:108]:
```suggestion
- Added debugging support for package dependency sources. With this feature, now, you can step in and debug the sources
```

File: release-notes/swan-lake-alpha4-release-note.md [hunk 3]
```diff
@@ -105,6 +105,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
...
+  related to the following dependency package types, that are used within your ballerina program.
```
Comment (praneesha) [inline release-notes/swan-lake-alpha4-release-note.md:109]:
```suggestion
  related to the following dependency package types, which are used within your Ballerina program.
```

File: release-notes/swan-lake-alpha4-release-note.md (hunk 1 above)
Comment (praneesha) [inline release-notes/swan-lake-alpha4-release-note.md:112]:
```suggestion
  - imported modules from Ballerina Central
```

Comment (NipunaRanasinghe) [reply release-notes/swan-lake-alpha4-release-note.md:112]: @praneesha WDYT?

Comment (MaryamZi) [reply release-notes/swan-lake-alpha4-release-note.md]: +1

**Review by praneesha — APPROVED** [review]

**Pull request merged**